pkgs/by-name/ma/material-maker/package.nix

```
@@ -0,0 +1,138 @@
+{
+  lib,
+  stdenv,
+  fetchzip,
+  fetchurl,
+  nix-update-script,
+  pkgs,
+
+  # build deps
+  autoPatchelfHook,
+  copyDesktopItems,
+  makeDesktopItem,
```

There is actually a setup hook for unpacking `dmg` files

```suggestion
  makeDesktopItem,
  undmg,
```